test/views/agent/partials/VatReturnsPartialSpec.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.agent.partials
+
+import assets.messages.partials.{VatReturnsPartialMessages => Messages}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.mvc.AnyContentAsEmpty
+import play.api.test.FakeRequest
+import views.ViewBaseSpec
+
+class VatReturnsPartialSpec extends ViewBaseSpec {
+
+  "VatReturnsPartial" should {
```

Comment:
```suggestion
  "VatReturnsPartial" when {
```
'When' makes more sense here as you're following it with two different scenarios. It then makes the output of the test more readable